app/assets/javascripts/dialog_field_refresh.js

```diff
@@ -1,16 +1,20 @@
 /* global miqInitSelectPicker miqSelectPickerEvent miqSparkle miqSparkleOn */
 
 var dialogFieldRefresh = {
+  unbindAllPreviousListeners: function() {
+    $(document).off('autoRefresh');
```

Comment:
@syncrou Correct, instead of just a generic 'message' (which is how the `postMessage` call was working), I just made it so the trigger event name is 'autoRefresh'. We could even be more specific if you wanted and make it something like 'dialog::autoRefresh' or whatever if you have a better idea or a clearer name, but this is just how jQuery does it and allows us to use the `.off` method to unbind all the current listeners.